src/common/explanation/Explanation.tsx

```diff
@@ -30,7 +38,14 @@ function Explanation({ className, main, small, variant = 'margin' }: Props) {
         className
       )}
     >
-      <h1 className={styles.main}>{main}</h1>
+      <h1
```

Comment:
We really need to get the header sizes upstream in HDS sorted!!

Reply:
Maybe we can add some kind of TODO here so it is easy to find the spots that needs fixing once it happens..

Reply:
I created an issue in HDS: https://github.com/City-of-Helsinki/helsinki-design-system/issues/143

Reply:
And maybe this is not the best example of this issue.

Reply:
Yeah, I think that we may still have some things to work out within this app 🤔 I'm not sure if all the H1 element we have actually need to be H1 elements. Then some of the H1 element we have are smaller than some other visual headings we have. But yeah, I wasn't able to get my head around it enough that I would have liked to make the change here.